components/measurement/SummaryText.js

```diff
@@ -14,8 +14,16 @@ const SummaryText = ({
   content,
 }) => {
   const metadata = getTestMetadata(testName)
-  const formattedDate = moment.utc(date).format('LL')
-  const formattedDateTime = moment.utc(date).format('lll')
+  const formattedDate = moment(date).format('LL')
+  const formattedDateTime = intl.formatDate(moment.utc(date).toDate(), {
+    year: 'numeric',
+    month: 'long',
+    day: '2-digit',
+    hour: '2-digit',
+    minute: 'numeric',
+    timeZone: 'UTC',
+    timeZoneName: 'short'
+  })
```

Comment:
@sarathms is this way of displaying dates consistent with how we do it in other places?

Reply:
This actually follows the way @sarathms did in https://github.com/ooni/explorer/commit/e0ee6147a530ec1f09c95807d88c74ffc2ad55b8. I followed that